### PR TITLE
Add optional field for project Twitter url, refs #329.

### PR DESCRIPTION
**Pull request by juliaelman**
This pull request adds a migration as well as a template update. The template will check first for the project Twitter handle and then the organization.

### CODE REVIEW

**Comment by DBNess** [comment]
:+1: niiiice, thank you, @juliaelman!

**Comment by courte** [comment]
@juliaelman - I checked this out locally and the database and front end layers look great!

Would you be able to update the admin areas as well (`/app/admin/projects.rb`)?

ActiveAdmin can be confusing if you haven't seen it before. Let me know if you have any questions. :)

**Comment by juliaelman** [comment]
@courte updates made! Let me know if I need to change/update anything (pretty sure I did it right...).

**Comment by courte** [comment]
Just one thing missing, @juliaelman. The `:twitter` attribute needs to be added to the `attr_accessible` list  in Project class (`app/models/project.rb`) so it can be mass-assigned in ActiveAdmin. Once it's got that, it should be :100:!

**Comment by juliaelman** [comment]
@courte done!

File: app/admin/projects.rb [hunk 1]
```diff
@@ -86,6 +88,9 @@
       row 'Approved?' do ad.is_approved end
       row :created_at
       row :updated_at
+      row :twitter_url do |proj|
+        link_to proj.twitter, proj.twitter, target: "blank" if proj.twitter?
```
Comment (courte) [inline app/admin/projects.rb]: Right now the link href is set to proj.twitter. You'll need to insert an `"https://twitter.com/" + ` into the second argument to get this going where you'd expect.

This is the last, last thing. Probably. :)

**Comment by DBNess** [comment]
:+1::+1::+1::sunglasses:

**Pull request merged**